scripts/helpers.js

```diff
@@ -129,6 +142,62 @@ hexo.extend.helper.register('docs_url_prefix', function (item) {
   return this.page.source.split('/', 2).join('/') + '/';
 });
 
+/**
+ * Return actively-being-viewed version of the docs based on page.
+ * Extract <VERSION> out of `.../docs/<VERSION>/...`.
+ */
+hexo.extend.helper.register('docs_active_version', function (page) {
+  var splitPath = page.path.split('/');
+  var docIndex = splitPath.indexOf('docs');
+  return splitPath[docIndex + 1];
+});
+
+/**
+ * Sort documentation pages. Pages that have `order` go first. Then alphabetized by title.
+ */
+hexo.extend.helper.register('docs_nav_sort', function (pages) {
+  var orderedPages = [];
+  var alphabetizedPages = [];
+  pages.forEach(function addOrderedPages (page) {
+    if ('order' in page) {
+      orderedPages.push(page);
+      return;
+    }
+    alphabetizedPages.push(page);
+  });
+  orderedPages = orderedPages.sort(function sort (pageA, pageB) {
+    if (pageA.order < pageB.order) { return -1; }
+    if (pageA.order > pageB.order) { return 1; }
+    return 0;
+  });
+  alphabetizedPages = alphabetizedPages.sort(function sort (pageA, pageB) {
+    if (pageA.title < pageB.title) { return -1; }
+    if (pageA.title > pageB.title) { return 1; }
+    return 0;
+  });
+  return orderedPages.concat(alphabetizedPages);
+});
+
+/**
+ * Filter documentation navigation to only include pages of the version being browsed.
+ */
+hexo.extend.helper.register('docs_version_filter', function (pages, version) {
+  return pages.filter(function (page) {
+    return page.path.indexOf('docs/' + version) !== -1;
+  });
+});
+
+/**
+ * Return list of A-Frame versions, including `master`.
+ * Order: current, master, <oldVersions>.
+ */
+hexo.extend.helper.register('docs_versions', function (currentVersion) {
+  var versions = aframeVersions.slice(0);
+  var currentVersionIndex = versions.indexOf(currentVersion);
+  versions.splice(currentVersionIndex, 1);
+  return [currentVersion, MASTER].concat(versions);
```

Comment:
it seems odd to me that `MASTER` appears _below_ not above `0.2.0`? don't we want to sort chronologically?

![image](https://cloud.githubusercontent.com/assets/203725/15693226/71e40516-2748-11e6-8ee1-5d0a2c6d548c.png)

can we change that?